packages/quantic/force-app/main/default/lwc/quanticFacetManager/quanticFacetManager.js

```diff
@@ -129,7 +129,6 @@ export default class QuanticFacetManager extends LightningElement {
 
   resolveItemTemplate() {
     this.itemTemplate = document.createElement('div');
-    this.itemTemplate.classList.add('slds-var-m-bottom_large');
```

Comment:
removed the margin here since we are changing the templates directly in the HTML to support both DNE enabled/disabled